### DISCUSSION TITLE
Quasar list element not wrapping vertically

### DISCUSSION THREAD

**Discussion by Marecu** [quasarframework/quasar#9699] [General - Components / Directives / etc] [answered]
Very new to Quasar so sorry if this is stupid - 

I'm trying to create a list that displays the desired content in a column and then wraps vertically to keep the list going in another column once the first column has filled the page vertically. The problem I'm having is that the list will just continue to add elements in the first column and going off the page instead of wrapping it. 
This is what the list looks like now, the list continues down off the page:
![image](https://user-images.githubusercontent.com/82179952/122287585-54f66f80-ceae-11eb-8f9d-ef7bbb0c7b92.png)
My code for the section is here:
`<div class="column">`
   `  <q-list>`
    `    <div v-for="(item, index) in array" :key="index" class="row justify-start items-center">`
    `      <q-btn color="positive" @click="additem(index, item)">+</q-btn>`
    `      <q-btn color="negative" @click="removeitem(index, item)">-</q-btn>`
  `        {{ item.text }} x {{ item.quantity}}`
  `      </div>`
   `   </q-list>`
 `   </div>`
To my understanding based on this page ([https://quasar.dev/layout/grid/introduction-to-flexbox](url)), the parent div should automatically wrap the component divs vertically once the page fills. Instead, it doesn't wrap, so I'm clearly doing something wrong. What am I missing here? Any help would be appreciated :)

**Comment by ontwikkelfabriek** [comment, marked as answer]
@Marecu 

If you want the best support. Create a codepen.io that demo's your issue. You can easily create a codepen bij extending on of the quasar examples or use this default empty one:
https://codepen.io/rstoenescu/pen/VgQbdx

A quick look at you code, 
1)  the **direct** child of the element that has the `column` ( or `row`) class must have a `col` class.
2) learn flexbox, because what you want is not 'hello world' in flex box

```
<div >
  <q-list class="column justify-start items-center">
    <div v-for="(item, index) in array" :key="index" class="col">
      <q-btn color="positive" @click="additem(index, item)">+</q-btn>
      <q-btn color="negative" @click="removeitem(index, item)">-</q-btn>
       {{ item.text }} x {{ item.quantity}}
    </div>
  </q-list>
</div>
```
